### DISCUSSION TITLE
Afk dont get attacked my mobs

### DISCUSSION THREAD

**Discussion by Tizooo** [PurpurMC/Purpur#519] [__Completed__] [answered]
make it so AFk people wont get attacked by mobs since i have autorespawn on my server is sometimes spams my server if there is a afk player which is getting killed 24/7 by mobs

**Comment by glenngenre** [comment]
Wouldn't your players just do /afk when they're about to die?

**Reply by Tizooo** [reply]
there is no afk command they only get marked as afk if they stand still for 1min

**Comment by granny** [comment, marked as answer]
Implemented as of this commit https://github.com/pl3xgaming/Purpur/commit/fc79bca3c57432c21119120b0252020103b59402
